### PR TITLE
Added Hindi translations.

### PR DESCRIPTION
**Pull request by devendrapoonia**

### CODE REVIEW

File: src/main/resources/assets/seedmapper/lang/hi_in.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  "command.clearscreen.empty": "कोई रेंडर हटा नहीं सका।",
+  "command.clearscreen.success": "%d रेंडर हटा दिए गए।",
+
+  "command.config.setAutomate": "Set \"automate\" to %s.",
+  "command.config.getAutomate": "\"automate\" is set to %s.",
+  "command.config.toggleAutomate": "Toggled \"automate\" to %s.",
+  "command.config.setSeed": "Set \"seed\" to %d.",
+  "command.config.getSeed": "\"seed\" is set to %d.",
+  "command.config.getSeed.null": "\"seed\" is null.",
```
Comment (xpple) [inline src/main/resources/assets/seedmapper/lang/hi_in.json:35]: Why aren't these translated?

**Review by xpple — CHANGES_REQUESTED** [review]
Thanks! Just one minor thing.

File: src/main/resources/assets/seedmapper/lang/hi_in.json (hunk 1 above)
Comment (devendrapoonia) [reply src/main/resources/assets/seedmapper/lang/hi_in.json:35]: Translating these was kind of hard and not really making any sense, some languages have not translated these too.

Comment (xpple) [reply src/main/resources/assets/seedmapper/lang/hi_in.json:35]: What precisely is hard about translating these sentences?

File: src/main/resources/assets/seedmapper/lang/hi_in.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  "commands.exceptions.versionNotFound": "संस्करण \"%s\" नही मिला",
+  "commands.exceptions.unknownConfig": "अज्ञात कॉन्फिग \"%s\".",
+  "commands.exceptions.nullPointerException": "\"%s\" null है",
+  "commands.exceptions.invalidDimension": "अमान्य आयाम।",
+  "commands.exceptions.biomeNotFound": "बायोम \"%s\" नही मिली।",
+  "commands.exceptions.structureNotFound": "सरंचना \"%s\" नही मिली",
+  "commands.exceptions.lootNotFound": "आइटम \"%s\" नही मिला",
+  "commands.exceptions.lootItemNotFound": "लूट आइटम \"%s\" नही मिला।",
+  "commands.exceptions.blockNotFound": "ब्लॉक \"%s\" नही मिला।",
+  "commands.exceptions.expectedWithOrWithout": "Expected \"with\" or \"without\".",
```
Comment (xpple) [inline src/main/resources/assets/seedmapper/lang/hi_in.json:12]: If you could translate this one as well that would be amazing :).

**Review by xpple — CHANGES_REQUESTED** [review]
Thanks!

File: src/main/resources/assets/seedmapper/lang/hi_in.json (hunk 2 above)
Comment (devendrapoonia) [reply src/main/resources/assets/seedmapper/lang/hi_in.json]: Done :D

**Review by xpple — APPROVED** [review]

**Pull request merged**